docs/options.md

```diff
@@ -163,3 +163,33 @@ To change this behaviour, following options are provided:
 * `--allocate-determ-start-addres`     - Controls the required start address of the pre-allocated memory. This address has to be page aligned. If null is provided, the memory is mapped to an arbitrary address.
 * `--return-null-on-zero-malloc`       - Controls if a NULL pointer should be returned in case the size argument is zero (*default=off*)
 * `--red-zone-space`                   - Controls the space kept unused between two adjacent allocations in byte (*default=10*)
+
+## Making KLEE Exit on Events
+
+KLEE does not implicitly exit on hit assertion or errors. On the other hand, it implicitly exits when solver fails for some reason.
```

Comment:
KLEE doesn't exit implicitly. You might want to say: "KLEE does not exit if a bug is found in the analyzed application" which is typically the main focus of SE, to find as much bugs as possible.
The "solver fail" is the inability of KLEE to check the current program state because of limitations of the solver.

But thinking about it, maybe we should indeed integrate the solver part as well.
As it is a limitation, similar to the POSIX layer, memory model, ... .


"assertion or errors" -> "assertions or errors".